internal/api/registry/manifests.go

```diff
@@ -198,8 +198,8 @@ func (a *API) handleGetOrHeadManifest(w http.ResponseWriter, r *http.Request) {
 		w.Write(manifestBytes)
 	}
 
-	//count the pull
-	if r.Method == http.MethodGet && r.Header.Get("X-Keppel-No-Count-Towards-Last-Pulled") != "1" {
+	//count the pull unless a special header is set or the token was issued by the janitor user identity used by the trivy proxy
+	if r.Method == http.MethodGet && r.Header.Get("X-Keppel-No-Count-Towards-Last-Pulled") != "1" && authz.UserIdentity.PluginTypeID() != "janitor" {
```

Comment:
I don't want to introduce dependencies on these plugin type IDs unless absolutely necessary. We have the UserType enum for recognizing types of users like this:
```suggestion
	if r.Method == http.MethodGet && r.Header.Get("X-Keppel-No-Count-Towards-Last-Pulled") != "1" && authz.UserIdentity.UserType() != keppel.JanitorUser {
```